db/migrate/20180904135235_create_campaigns.rb

```diff
@@ -0,0 +1,11 @@
+class CreateCampaigns < ActiveRecord::Migration[5.2]
+  def change
+    create_table :campaigns do |t|
+      t.string :title
+      t.text :description
+      t.string :location
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
This migration is also part of another PR. Why are you doing this in this sequence. Surely there must be other stories you can work on without editing the same thing twice and generate conflicts. You only have 2 weeks to work on this project.